atlasdb-client/src/main/java/com/palantir/atlasdb/stream/AbstractGenericStreamStore.java

```diff
@@ -225,7 +214,8 @@ private void tryWriteStreamToFile(Transaction transaction, T id, StreamMetadata
 
     protected abstract File createTempFile(T id) throws IOException;
 
-    protected abstract void loadSingleBlockToOutputStream(Transaction tx, T streamId, long blockId, OutputStream os);
```

Comment:
This is fine. I don't know if the large internal product has this lined up appropriately to auto-regenerate their schemas, so we might need to do something if it doesn't